### PR TITLE
[CPDLP-1236] Add ECF statement selector to finance area

### PR DESCRIPTION
**Pull request by asmega**
## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDLP-1236

- Added statement selector to ECF statements in finance area
- Also tweaked the top area of the page to match the designs more

## Screenshots

Screenshot 2022-04-13 at 09 40 01

### Code quality checks
- [x] All commit messages are meaningful and true
- [x] Added enough automated tests

### HTML Checks
- [x] All new pages have automated accessibility checks
- [x] All new pages have visual tests via Percy

### Gotchas
- [x] All `School` queries are correctly scoped - `eligible` when they need to be

### How can someone see it it review app?

- Login as a finance user
- View an ECF statement
- Should be able to change statements now via the statement selector component

## External API changes

- None

### CODE REVIEW

File: app/components/finance/statements/ecf_statement_selector.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    statements,
+    :id,
+    :name,
+    label: { text: "View a different statement" },
+    options: { selected: current_statement.name.parameterize },
+    form_group: { class: "govuk-form-group govuk-!-width-one-third" }
+  %>
+
+  <%= f.govuk_submit "View", secondary: true %>
+<% end %>
```
Comment (ericaporter) [inline app/components/finance/statements/ecf_statement_selector.html.erb:21]: Could add text to en.yml if it isn't already there

File: app/views/finance/ecf/statements/show.html.erb [hunk 2]
```diff
@@ -2,20 +2,18 @@
...
-    <ul class="govuk-list">
-      <% @statements.each do |statement| %>
-        <li>
-          <%= text_otherwise_link_to(statement_display_text(statement), finance_ecf_payment_breakdown_statement_path(@ecf_lead_provider, statement), statement == @statement) %>
-        </li>
-      <% end %>
-    </ul>
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-xl">Early career framework (ECF)</h1>
+
```
Comment (ericaporter) [inline app/views/finance/ecf/statements/show.html.erb:7]: Could add heading to en.yml

Comment (asmega) [reply app/views/finance/ecf/statements/show.html.erb:7]: I've extracted the h1 copy to an i18n string

File: app/components/finance/statements/ecf_statement_selector.html.erb (hunk 1 above)
Comment (asmega) [reply app/components/finance/statements/ecf_statement_selector.html.erb:21]: I've extracted the submit button to an i18n string

**Review by ericaporter — APPROVED** [review]

**Pull request merged**